### PR TITLE
Explicitly mention that Metalk8s 1.x is no longer maintained

### PR DESCRIPTION
**Pull request by sayf-eddine-scality**
**Component**:
Docs

**Context**: 
Metalk8s 1.x
**Summary**:
v 1.x is no longer maintained, better mention it

### CODE REVIEW

**Review by gdemonet — APPROVED** [review]
Good catch, thanks @sayf-eddine-scality!

File: README.md [hunk 1]
```diff
@@ -70,7 +70,5 @@ tox -e docs
 
 ---
 
-MetalK8s version 1 is still maintained in this repository. See the
-`development/1.*` branches, e. g.
-[MetalK8s 1.3](https://github.com/scality/metalk8s/tree/development/1.3) in the
-same repository.
+MetalK8s version 1 is still hosted in this repository but is no longer maintained.
+The last release is [MetalK8s 1.3](https://github.com/scality/metalk8s/tree/development/1.3).
```
Comment (NicolasT) [inline README.md:74]: 1.3 isn't strictly a release... Also, not sure we should talk about 1.x at all in this document anymore, at all.

Comment (sayf-eddine-scality) [reply README.md:74]: ok will remove it all

**Pull request merged**